.github/workflows/build.yml

```diff
@@ -148,7 +146,17 @@ jobs:
         shell: pwsh
 
       - name: Run Core tests
```

Comment:
ℹ️ I don't feel that this should be within the Android build vs. something fundamental across all builds earlier but given the significant setup per platform I am gonna leave it be for now since the results will still appear regardless.

Reply:
DevOps hasn't done a whole lot with the mobile build pipeline over the past two years. Mobile has been a large growth area for DevOps that we have yet to have the capacity to grow in and improve the processes around.

I'd love to hear some thoughts on what we can improve and how to improve.